app/controllers/product_review_videos/streams_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class ProductReviewVideos::StreamsController < ApplicationController
+  before_action :set_product_review_video
+
+  def show
+    respond_to do |format|
+      format.smil { render plain: @product_review_video.video_file.smil_xml }
```

Comment:
This and the related implementation follows 

https://github.com/antiwork/gumroad/blob/ff5ec881e28c0d59239dacb14f124271320a777a/app/controllers/url_redirects_controller.rb#L120-L125